src/Giveaway.js

```diff
@@ -308,6 +319,34 @@ class Giveaway extends EventEmitter {
         return true;
     }
 
+    /**
+     * @param {Discord.User} user The user to check
+     * @returns {Promise<number|boolean>} The highest bonus entries the user should get or false
+     */
+    async checkBonusEntries(user) {
+        const member = this.channel.guild.member(user.id);
+        const entries = [];
+
+        if (this.bonusEntries.length) {
+            for (const obj of this.bonusEntries) {
+                const filter = obj[Object.keys(obj)[0]];
+                const bonus = obj[Object.keys(obj)[1]];
```

Comment:
so here we should use properties instead of Object.keys

Reply:
Changed 👍